src/main/java/nl/hsac/fitnesse/fixture/slim/web/RichFacesTest.java

```diff
@@ -145,7 +145,7 @@ protected void storeLocationBeforeAction() {
 
     protected boolean eventTriggersAjax(WebElement element, String attribute) {
         String eventHandler = element.getAttribute(attribute);
-        return eventHandler != null && eventHandler.startsWith("RichFaces.ajax(");
+        return eventHandler != null && eventHandler.contains("RichFaces.ajax(");

```

Comment:
I did not do this before because if the attribute does not start with the call we cannot be sure that the Ajax call will actually be executed (the might be an `if` before it causing no Ajax call to be sent). And in that case waiting for completion of a call that is not sent will not end so it will break the test.

Is that something that cannot happen, so an unfounded fear?